exercises/triangle/example.go

```diff
@@ -26,9 +26,11 @@ func KindFromSides(a, b, c float64) Kind {
 	}
 	// sides are now sorted
 	switch {
+	case math.IsInf(c, 0):
```

Comment:
someone might be confused about why you only need to check `c` and not `a`, `b`. comment?